tensorflow/python/ops/logging_ops.py

```diff
@@ -72,13 +72,10 @@
                           "only a concern in graph mode. Below is an example "
                           "of how to ensure tf.print executes in graph mode:\n"
                           """```python
-    sess = tf.Session()
-    with sess.as_default():
-        tensor = tf.range(10)
-        print_op = tf.print(tensor)
-        with tf.control_dependencies([print_op]):
-          out = tf.add(tensor, tensor)
-        sess.run(out)
+    tensor = tf.range(10)
```

Comment:
The deprecation message I think is fine using a tf.Session; it's giving directions to 1.x users about how to switch within 1.x.

Reply:
add comment as [`b38ef23`](https://github.com/tensorflow/tensorflow/pull/27479/commits/b38ef235d56a340cb080da328f47294ce24a92ec)